plugins/inputs/varnish/README.md

```diff
@@ -6,27 +6,44 @@ This plugin gathers stats from [Varnish HTTP Cache](https://varnish-cache.org/)
 
 ```toml
 [[inputs.varnish]]
-  ## If running as a restricted user you can prepend sudo for additional access:
-  #use_sudo = false
+## If running as a restricted user you can prepend sudo for additional access:
+#use_sudo = false
 
-  ## The default location of the varnishstat binary can be overridden with:
-  binary = "/usr/bin/varnishstat"
+## The default location of the varnishstat binary can be overridden with:
+binary = "/usr/bin/varnishstat"
 
-  ## By default, telegraf gather stats for 3 metric points.
-  ## Setting stats will override the defaults shown below.
-  ## Glob matching can be used, ie, stats = ["MAIN.*"]
-  ## stats may also be set to ["*"], which will collect all stats
-  stats = ["MAIN.cache_hit", "MAIN.cache_miss", "MAIN.uptime"]
+## Additional custom arguments for the varnishstat command
+# binary_args = ["-f", "MAIN.*"]
 
-  ## Optional name for the varnish instance (or working directory) to query
-  ## Usually append after -n in varnish cli
-  # instance_name = instanceName
+## The default location of the varnishadm binary can be overridden with:
+adm_binary = "/usr/bin/varnishadm"
 
-  ## Timeout for varnishstat command
-  # timeout = "1s"
+## Custom arguments for the varnishadm command 
+# adm_binary_args = [""] 
+
+## Metric version defaults to metric_version=1, use metric_version=2 for removal of nonactive vcls.
```

Comment:
Can we add something about the version of varnish which is supported by metric_version=2?

Reply:
Added varnish 6.0.2+ requirement into telegraf config example in README